doc/tests/manual/reports/230725.md

```diff
@@ -0,0 +1,292 @@
+
+## TC-009 - Timeline
+
+- Description: Change the date of the map to 'unhide' plantings.
+- Preconditions:
+  - User must be on the map planning screen.
+- Test Steps:
+  1. Add a planting to the map.
+  2. Click on the date selection on the bottom of the screen.
+  3. Navigate to a date in the past.
+  4. Wait 1 second.
+  5. Navigate to today.
+- Expected Result:
+  - The indicator was briefly blue, indicating a loading state.
+  - The indicator beside the input is green.
+  - The Date on the bottom/right corner of the screen shows the current day.
+  - The planting previously planted was gone while being in the past.
+  - The planting is visible again.
+- Actual Result:
+- Test Result: ⚠️
+- Notes: Issue [#699](https://github.com/ElektraInitiative/PermaplanT/issues/699), image loading error was present.
+
+## TC-010 - Timeline
+
+- Description: Change the `add_date` of a planting to 'hide' it.
+- Preconditions:
+  - User must be on the map planning screen.
+- Test Steps:
+  1. Add a planting to the map.
+  2. Click on the planting.
+  3. Click on the `Add Date` date selector in the left lower toolbar.
+  4. Change the date to a date in the future.
+- Expected Result:
+  - The indicator was briefly blue, indicating a loading state.
+  - The indicator beside the input is green.
+  - The planting previously planted is gone.
+- Actual Result:
+- Test Result: ✔️
+- Notes:
+
+## TC-011 - Timeline
+
+- Description: Change the `remove_date` of a planting to 'hide' it.
+- Preconditions:
+  - User must be on the map planning screen.
+- Test Steps:
+  1. Add a planting to the map.
+  2. Click on the planting.
+  3. Click on the `Remove Date` date selector in the left lower toolbar.
+  4. Change the date to today.
+- Expected Result:
+  - The indicator was briefly blue, indicating a loading state.
+  - The indicator beside the input is green.
+  - The planting previously planted is gone.
+- Actual Result: Error message: "Das Datum an dem die Pflanze hinzugefügt wurde muss vor dem Datum liegen, an dem sie entfernt wurde."
+- Test Result: ❌
+- Notes: When I change the planted day to yesterday and clear and reselect the remove date it works.
+  It seems to have and issue when add date and remove date are the same.
+
+## TC-012 - Timeline
+
+- Description: Change the `add_date` of a planting to 'unhide' it.
+- Preconditions:
+  - User must be on the map planning screen.
+- Test Steps:
+  1. Add a planting to the map.
+  2. Click on the planting.
+  3. Click on the `Add Date` date selector in the left lower toolbar.
+  4. Change the date to a date in the future.
+  5. Wait one second
+  6. Remove the date
+- Expected Result:
+  - The indicator was briefly blue, indicating a loading state.
+  - The indicator beside the input is green.
+  - The planting previously planted was gone while its `add_date` was in the future.
+  - The planting is visible now with its `add_date` unset.
+- Actual Result: The plant did not appear again.
+- Test Result: ❌
```

Comment:
I think there are even two bugs: one that unhiding without unselecting the plant does not work.
An the other one is that debouncing seems to be buggy: when you quickly click somewhere else, the removal or changing of date is not stored at all.

Can you create issues for this please and assign @Bushuo?

Reply:
With unhiding, do you mean changing the `add_date` to a `add_date <= current_date ||  add_date < null`?

For the second issue, it is not mentioned in #705. Where did you click? How can I reproduce this?

Reply:
Ah I see, you clicked on the map, which unselects the planting and thus unmounts the component.
I have to see how I can fix this...